Projects/Embark/Sources/EmbarkStore.swift

```diff
@@ -36,6 +36,10 @@ class EmbarkStore {
             }
         }
     }
+    
+    func getAllValues() -> [String:String] {
+        return revisions.last ?? [:]
```

Comment:
We would need to evaluate all computed values here before returning right? Also will we get any performance implications by that 🤔